interfaces/Portalicious/src/styles.scss

```diff
@@ -135,4 +135,22 @@ $toolbar-height: 48px;
       }
     }
   }
+
+  // Form inputs (Overrides for PrimeNG-components)
+  // All inputs used inside a form-field-wrapper, should be full-width, the wrapper should be in control of the width.
+  app-form-field-wrapper {
```

Comment:
I'm a bit concerned about using one of our custom component selectors here - this could change without us realising that we're breaking this. Can we apply these rules without the wrapper? If not, then this might be an exception where having a dedicated CSS file for a component (app-form-field-wrapper) might make sense.

Reply:
Yeah, I was a bit in doubt on how big I wanted the "impact area" to be...
This was my solution to have it work "opt in" (and keep the default styles available on other/future/yet-unknown use-cases)..

But the other approach could be indeed to "change the default" and deal with the scenario's of "but for this situation we need an exception" later...
The only thing is that currently I don't know the difference between a "one-off exceptional change" and a "this is how we always want it to be", so I try to stay on the (too?) cautious side.

Reply:
I saw the additions to `styles.scss`, including the re-org and new comments. They look great, thanks! ✨